packages/shared/src/components/post/PostContent.tsx

```diff
@@ -71,6 +71,7 @@ export function PostContentRaw({
     'laptop:flex-row laptop:pb-0',
     className?.container,
   );
+  useSharedByToast();
```

Comment:
Can't we just do this on the post page?
Then you only need to do it once.

We did the rendering in all 3 because it's a render thing.